src/js/announcements.jsx

```diff
@@ -29,7 +29,7 @@ class Announcements extends React.Component {
   render() {
     const { announcements } = this.state;
     return (
-      <div id="announcements">
+      <div id="announcements" class="section">
```

Comment:
I think you meant `className` here (and a few other places)?

Reply:
Good catch, thanks